src/components/map/Mapbox.vue

```diff
@@ -1,37 +1,54 @@
 <!-- Landing page (using mapbox) -->
 <template>
-  <MglMap
-    container="mapbox-container"
-    :access-token="accessToken"
-    :map-style="mapStyle"
-    :center="defaultMapStyle.center"
-    :zoom="defaultMapStyle.zoom"
-    @load="onMapLoaded"
-  >
-    <MapMarker
-      v-for="(event, index) in events"
-      :key="`marker-${index}`"
-      :name="event.name"
-      :coords="event.venue.location.coordinates"
-      :img-src="event.media.coverPhoto.baseSrc"
-      @navigate="navigateToEvent(event)"
-    />
-  </MglMap>
+  <div>
+    <MglMap
+      id="map"
+      container="mapbox-container"
+      :access-token="accessToken"
+      :map-style="mapStyle"
+      :center="defaultMapStyle.center"
+      :zoom="defaultMapStyle.zoom"
+      @load="onMapLoaded"
+    >
+      <MapMarker
+        v-for="(event, index) in events"
+        :key="`marker-${index}`"
+        :name="event.name"
+        :coords="event.venue.location.coordinates"
+        :img-src="event.media.coverPhoto.baseSrc"
+        @navigate="navigateToEvent(event)"
+      />
+      <!-- <img src="@/assets/icons/search.png" alt class="search-icon" /> -->
+      <MglGeocoderControl
+        v-model="searchQuery"
+        :access-token="accessToken"
+        class="map-search"
+        @results="handleSearch"
+      />
+    </MglMap>
+    <!-- <SearchBar id="searchbar" @update:query="updateQuery" /> -->
```

Comment:
Let's deprecate this component now shall we

Reply:
We still need this component for searching in the events dashboard

Reply:
Okk then lets just remove from mapbox component